test/javascripts/lib/utilities-test.js.es6

```diff
@@ -186,3 +187,41 @@ QUnit.test("fillMissingDates", assert => {
     "it returns a JSON array with 31 dates"
   );
 });
+
+QUnit.test("inCodeBlock", assert => {
```

Comment:
super nice to have this test and nice specification. I wonder though if we should make it a bit more readable?

`test V'X'`... use V to denote that auto complete will open and X to denote it will not, then you zoom through the string and create assertions on the fly. Way easier to reason about the implementation.

Reply:
The test was built kind of like that. `foo` was always outside of a code block and `bar` was always inside a code block. I replaced it with "0" (not in code block) and "1" (in code block) and am generating the assertions on-the-fly now.

I am not a big fan of generating assertions because then I feel like the test contains too much logic. I do agree though that it is a lot more readable.